integration/run/run_test.go

```diff
@@ -1192,7 +1497,10 @@ func TestUsingComputeClasses(t *testing.T) {
 				}
 
 				// Assign a name for the test case so no collisions occur
-				app, err := c.AppRun(ctx, image.ID, &client.AppRunOptions{Name: testcase})
+				app, err := c.AppRun(ctx, image.ID, &client.AppRunOptions{
+					Name:   testcase,
+					Region: apiv1.LocalRegion,
```

Comment:
Why is this changing? We should be able to continue to use the default region here instead of specifying one, correct?

Reply:
Yes, that was left over from previous versions of this PR, I've reverted it.